utils/hudi_clickhouse_sync.py

```
@@ -60,93 +62,126 @@ def read_hudi_table(self, table_name, base_path=None):
                 if date_str:
                     try:
-                        # Parse date string like "23-DEC-2025" 
-                        timestamp = datetime.strptime(date_str, "%d-%b-%Y")
-                    except ValueError:
-                        # If parsing fails, use current time
-                        timestamp = datetime.now()
-                
-                # Convert trade_date string to date
-                trade_date_str = str(row['trade_date']) if row['trade_date'] else None
+                        ts = datetime.strptime(str(date_str), "%d-%b-%Y")
+                    except Exception:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)